static/app/components/modals/importDashboardFromFileModal.tsx

```diff
@@ -0,0 +1,127 @@
+import {Fragment, useState} from 'react';
```

Comment:
I didn't write any specs for this component-- partly because it's an internal feature, partly because I didn't want to invest time in it if it wasn't necessary here. Let me know if I should.